js/minirepl.js

```diff
@@ -109,7 +109,7 @@ function compileCode(sourceEditor, targetEditor) {
 
   try {
     transformed = Babel.transform(sourceEditor.getValue(), {
-      presets: ["latest"],
+      presets: ["es2015", "es2016", "es2017"],
```

Comment:
we should would want to get preset-env to be default eventually right?

Reply:
I didn't want to load two different bundles just for the minirepl.
I think that, since it is just a demo, it is ok like this?

Reply:
totally didn't see it was the minirepl, makes sense!